src/main/scala/castalia/StatusService.scala

```diff
@@ -2,20 +2,16 @@ package castalia
 
 import java.lang.management.ManagementFactory
 
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
 
-import scala.concurrent.duration._
-
-trait StatusService extends BaseService {
-  protected val serviceName = "my service"
-
-  val statusRoutes: Route = pathPrefix("status") {
-    handleRejections(totallyMissingHandler) {
-      get {
-        log.info("/status executed")
-        complete(Status(Duration(ManagementFactory.getRuntimeMXBean.getUptime, MILLISECONDS).toString()))
-      }
+class StatusService()(implicit val system: ActorSystem) extends Routes with Protocol {
```

Comment:
Is there a reason to also mix in `Protocol`, since it is already implied by `Routes`